src/components/ebay-filter-menu/component.ts

```diff
@@ -56,6 +63,16 @@ export default class extends MenuUtils<Input, MenuState> {
         this.lastTabIndexPosition = 0;
     }
 
+    handleSearch(e: InputEvent) {
+        this.state.searchTerm = (e.target as HTMLInputElement).value;
+        this.emit("search-change", this.state.searchTerm);
```

Comment:
Do we want to add an option to filter the results? Like we have with combobox

Reply:
We can also iterate on this if needed

Reply:
I think it's overengineering to add our own filtering option, see the issue description for the results of a discussion with @ianmcburnie